graspologic/version/version.txt

```diff
@@ -1 +1 @@
-
+0.1.0.dev20201008123355
```

Comment:
have been trying to avoid modifying this file. seems we still need a soln or recommendation for what people should do here @dwaynepryce if you have any ideas?

Reply:
I suggested adding it to gitignore before and dont see it causing any problems. @dwaynepryce ?

Reply:
@loftusa  can you just remove this change for now

Reply:
a117443